.buildkite/pipeline.yml

```diff
@@ -68,10 +68,38 @@ steps:
     command: "goss validate --format documentation"
     timeout_in_minutes: 5
     agents:
-      stack: "buildkite-aws-stack-test-linux-${BUILDKITE_BUILD_NUMBER}"
-      queue: "testqueue-linux-${BUILDKITE_BUILD_NUMBER}"
+      stack: "buildkite-aws-stack-test-linux-amd64-${BUILDKITE_BUILD_NUMBER}"
+      queue: "testqueue-linux-amd64-${BUILDKITE_BUILD_NUMBER}"
     depends_on: "linux-launch"
 
+  - id: "packer-linux-arm64"
+    name: ":packer: :linux: 💪"
+    command: .buildkite/steps/packer.sh linux arm64
+    timeout_in_minutes: 60
+    retry: { automatic: { limit: 3 } }
+    agents:
+      queue: "${BUILDKITE_AGENT_META_DATA_QUEUE}"
+    depends_on:
+      - "lint"
+      - "bats-tests"
+
+  - id: "linux-arm64-launch"
+    name: ":cloudformation: :linux: 💪 Launch"
+    command: .buildkite/steps/launch.sh linux arm64
+    agents:
+      queue: "${BUILDKITE_AGENT_META_DATA_QUEUE}"
+    artifact_paths: "build/aws-stack.yml"
+    depends_on: "packer-linux-arm64"
+
+  - id: "linux-arm64-test"
+    name: ":cloudformation: :linux: 💪 Test"
+    command: "goss validate --format documentation"
+    timeout_in_minutes: 5
+    agents:
+      stack: "buildkite-aws-stack-test-linux-arm64-${BUILDKITE_BUILD_NUMBER}"
+      queue: "testqueue-linux-arm64-${BUILDKITE_BUILD_NUMBER}"
+    depends_on: "linux-arm64-launch"
```

Comment:
the build DAG now looks like this:

![Screenshot from 2020-10-30 11-42-41](https://user-images.githubusercontent.com/8132/97646915-1342c100-1aa5-11eb-8087-984f4a14cfaf.png)